### ISSUE TITLE
Import of components server side containing `svg` breaks production build

### ISSUE THREAD

**Issue by jaybe78** [brillout/vite-plugin-ssr#353]
## Context

My `page` components contain an `initData` function that needs to be called `onBeforeRender` to populate all the data.
In order to do that, I get the current url and based on that do a dynamic import of the page component to execute the `initData`.

## Error details
If the imported page component, contain import of `svg` file, for some reason the `ssr` plugins appends `extractStyles&lang.${fileExtension}`.

That result in the product build failing because the `svg` file cannot be found anymore.
The only workaround for now is to duplicate `svgs` with that extra string.

After investigating it seems `extractStylesAddQuery` is responsible for adding that string to `svg`
```
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.extractStylesAddQuery = void 0;
const utils_1 = require("../../utils");
function extractStylesAddQuery(id) {
    const fileExtension = (0, utils_1.getFileExtension)(id);
    (0, utils_1.assert)(fileExtension);
    return `${id}?extractStyles&lang.${fileExtension}`;
}
```

## How to reproduce the error

Please pull https://github.com/jaybe78/vps-webpack5-redux-streaming-test/tree/jb/bug/import-server-side-breaks-svg
then `npm run prod`

### Error Stack

```
[vite-plugin-svgr] ENOENT: no such file or directory, open '/Users/jb/Documents/vps-webpack5-redux-streaming-test/assets/svgs/button-app-store.svg?extractStyles&lang.svg'
```

**Comment by brillout** [comment]
Thanks for opening the issue. I will have a look at it later today.

**Comment by brillout** [comment]
Fix released in `0.4.0-beta.44`.
